network.go

```diff
@@ -0,0 +1,16 @@
+package sprig
+
+import (
+	"fmt"
+	"math/rand"
+	"net"
+)
+
+func getHostByName(name string) (string, error) {
```

Comment:
I'm sorry I didn't catch this before but returning an error has some consequences. It stops execution of the template where it is at and returns an error from `Execute`. You can see an example with this function at https://play.golang.org/p/d1f0BridbDc.

For sprig v3 we are going to add error handling via #132 and expanding it to cover any others.

Would you mind ignoring the error for now and we'll add that back when we have a consistent setup for both with and without error handling at that level.

For what it's worth, v3 is coming this month.

Reply:
Sure no problem at all!! :)

Reply:
I'll remove error catching from the test as well